packages/base-styles/_mixins.scss

```diff
@@ -489,12 +489,9 @@
 	/* stylelint-enable function-comma-space-after */
 }
 
-@mixin custom-scrollbars-on-hover() {
+@mixin custom-scrollbars-on-hover($handle-color, $track-color) {
```

Comment:
I moved this change to a separate PR as it will enable us to share this mixin elsewhere: https://github.com/WordPress/gutenberg/pull/49892